### DISCUSSION TITLE
I am not able to add GITHUB_USERNAME on secret  repository charu991/rolesix

### DISCUSSION THREAD

**Discussion by charu991** [community/community#64206] [Actions]
### Select Topic Area

Bug

### Body

Failed to add secret. Secret names must not start with GITHUB_.

please help me here.

**Comment by MaheshBabu11** [comment]
you can use the env variable ```GITHUB_ACTOR``` to get the user name if that's what you are looking for 

check this page for more info: https://docs.github.com/en/actions/learn-github-actions/variables
